src/neural/dx/fp16_utils.cc

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Rename this file to match .h ?

Reply:
done. created a separate header for those functions.